### ISSUE TITLE
[Bug] [a11y]PB-Add to homescreen-Text contrast needs to be changed for "Cancel" and "Add"

### ISSUE THREAD

**Issue by Diana-Rus** [mozilla-mobile/fenix#9656]
## Prerequisites:
- Have Google Accessibility Scanner installed.

### Steps to reproduce
1. Open app and from the homescreen, tap on "Private Browsing" icon.
2. Tap on the "Add Shortcut" button.
3. Scan the search screen.

### Expected behavior
There are no suggestions to make any changes.
### Actual behavior
Add to homescreen - "Cancel" button
- com.sec.android.app.launcher:id/add_item_cancel_button
The item's text contrast ratio is 2.91. This ratio is based on an estimated foreground color of #499EBD and an estimated background color of #FAFAFA.

Add to homescreen - "Add" button
- com.sec.android.app.launcher:id/add_item_add_button
The item's text contrast ratio is 2.91. This ratio is based on an estimated foreground color of #499EBD and an estimated background color of #FAFAFA.

### Device information

* Android device: Samsung Galaxy S9 (Android 8)
* Fenix version: Firefox Preview Nightly 4/3 (Build #20940609)

**Screens:**
**Add to homescreen - "Cancel" button**
![20200403_113815](https://user-images.githubusercontent.com/49859863/78342745-6baef000-75a2-11ea-8e55-82a87eb7aed1.jpg)

**Add to homescreen - "Add" button**
![Screenshot_20200403-113323_Accessibility Scanner](https://user-images.githubusercontent.com/49859863/78342760-6fdb0d80-75a2-11ea-98e0-a072f3597e18.jpg)

**Comment by asadotzler** [comment]
I've tested this with a Galaxy S10e running Android 10 and the Samsung One UI version 2.0 and I get a contrast ratio of 4.65 which is sufficient. I'm going to guess we're using a system color here and our different Android versions are showing different colors.

**Comment by mcarare** [comment]
Closing this, as it is a system dialog that we cannot improve.
